ansible/group_vars/do/main.yml

```diff
@@ -3,11 +3,13 @@ project_name: "default"
 
 main_node_size: "s-1vcpu-2gb"
 crawl_node_size: "c-4"
+admin_node_size: "s-4vcpu-8gb"
 droplet_region: "sfo3"
 
 node_pools:
   - name=main-app;size={{ main_node_size }};label=nodeType=main;count=2
   - name=crawling;size={{ crawl_node_size }};label=nodeType=crawling;taint=nodeType=crawling:NoSchedule;auto-scale=true;min-nodes=1;max-nodes=3;count=1
+  - name=admin-app;size={{ admin_node_size }};label=nodeType=admin;count=1
```

Comment:
Hm, do we want to also make this conditional, only add if `enable_admin_addons` is true?
Would need to modify Jinja2 template logic in the playbook that joins the array..

Reply:
Yes, right. That should be conditional. I will make some change for it.